dcrjson/parse.go

```diff
@@ -44,3 +47,66 @@ func DecodeConcatenatedHashes(hashes string) ([]chainhash.Hash, error) {
 	}
 	return decoded, nil
 }
+
+// EncodeConcatenatedVoteBits encodes both voteBits and extended voteBits into
+// a single slice for concatenation into a serialized byte slice.  The entirety
+// of the voteBits are encoded as follows:
+//
+//   Size            Description
+//   1 byte          Length of the concatenated voteBits in bytes
+//   2 bytes         Vote bits
+//   up to 73 bytes  Extended vote bits
+//
+// The result may be concatenated into a slice and then passed to callers
+func EncodeConcatenatedVoteBits(voteBits uint16, voteBitsExt []byte) ([]byte, error) {
```

Comment:
Also don't think we want to use `Concatenated` in the name here, since this is only dealing with a single `VoteBits` structure, not several of them concatenated together.